react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+let currentCount = 0;
+export default class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: currentCount,
+      isStarted: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.resetClick = this.resetClick.bind(this);
+  }
+
+  resetClick() {
+    if (this.state.isStarted === false) {
+      currentCount = 0;
+      this.setState({ count: currentCount });
+    }
+  }
+
+  handleClick() {
+    if (this.state.isStarted === false) {
+      this.setState({ isStarted: true });
+      this.intervalID = setInterval(() => this.setState({ count: currentCount++ }), 1000);
```

Comment:
I would originally declare this property in the constructor, and then update it as needed.